app/models/manageiq/providers/openstack/cloud_manager/host_aggregate.rb

```diff
@@ -1,8 +1,118 @@
 class ManageIQ::Providers::Openstack::CloudManager::HostAggregate < ::HostAggregate
+  include SupportsFeatureMixin
```

Comment:
no need to include it here again, because it's already included at the base class